Dockerfile

```diff
@@ -34,7 +34,9 @@ RUN /venv/bin/pip install -r https://raw.githubusercontent.com/mongodb/docs-tool
 RUN apt-get update && apt-get install -y python3 python3-dev python3-pip
 RUN apt-get -y install vim
 RUN apt-get -y install git pkg-config libxml2-dev
-RUN python3 -m pip install -r https://raw.githubusercontent.com/mongodb/mut/master/requirements.txt
+RUN python3 -m pip install https://github.com/mongodb/mut/releases/download/v0.10.1/mut-0.10.1-py3-none-any.whl
```

Comment:
Tasty.

There's the theoretical issue that we're no longer pinning dependencies, but I don't think we need to overthink this.

Reply:
Are they not semi-pinned in [pyproject.toml](https://github.com/mongodb/mut/blob/master/pyproject.toml) ?

Reply:
that's a dependency specification, not pinning per se. The idea of pinning is to express "developers have tested *this exact* dependency version and it is known to be good", preventing a regression in a dependency from hosing things.